INSTALL

```diff
@@ -1,575 +1,176 @@
+This file describes how to compile and install Freeciv21. Last time we made sure this file is up to date was 14-Feb-21. Last major
+update was 14-Feb-21.
 
 This document contains sections and subsections as follows:
      0. Prerequisites:
-     1. Prerequisites for the clients:
-          1a. Prerequisites for the Gtk3 client:
-          1b. Prerequisites for the Gtk3.22 client:
-          1c. Prerequisites for the SDL2 client:
-          1d. Prerequisites for the Qt client:
-     2. Generating Makefiles
-          2a. Generating the Makefile for git versions:
-          2b. Generating the Makefile for release versions:
-     3. Compiling Freeciv:
-     4. Installation:
-     5. Native Language Support:
-     6. Readline Support:
-     7. Reinier's Solaris Installation Notes:
-     8. Mac OS X and Darwin notes:
-     9. Windows notes:
+     1. Prerequisites for the Qt client:
+     2. Compiling and Installing Freeciv21:
+     3. Debian Notes:
+     4. Windows Notes:
 
 
 0. Prerequisites:
 =================
 
-Freeciv has a number of prerequisites.  Note, that apart from the first
-prerequisite, the Freeciv configuration process is smart enough to work
-out whether your system is suitable.  If in doubt, just try it.
+Freeciv21 has a number of prerequisites.  Note, that apart from the first prerequisite, the Freeciv21 configuration process is smart
+enough to work out whether your system is suitable.  If in doubt, just try it.
 
  - Unix (or similar)
 
-   The Unix operating system, a work-alike such as Linux or FreeBSD,
-   or some OS that provides a very Unix-like personality or mode, like
-   EMX under OS/2 or the Cygnus Cygwin toolkit under Windows.
-   (See http://sourceware.cygnus.com/cygwin/).  Support for
-   BSD-style TCP/IP sockets is essential, as is a Bourne-shell
-   compatible shell, such as GNU "bash".  (Most Unixes fit the bill...)
+The Unix operating system, a work-alike such as Linux or FreeBSD, or some OS that provides a very Unix-like personality or mode,
+like EMX under OS/2 or the Cygnus Cygwin toolkit under Windows.  (See https://www.cygwin.com/).  Support for BSD-style TCP/IP
+sockets is essential, as is a Bourne-shell compatible shell, such as GNU "bash".  (Most Unixes fit the bill...)
 
- - An ANSI C compiler.
+ - An ANSI C and C++ compiler.
 
-   Freeciv is written in very portable (almost) ANSI C.  Both 32- and 64-
-   bit machines are supported.  You cannot use a "K&R C" compiler, or
-   a C++ compiler.
+Freeciv21 is written in very portable (almost) ANSI C and C++.  Both 32- and 64-bit machines are supported.  You cannot use a "K&R
+C" compiler, or a C++ compiler.
 
-   Development of Freeciv is primarily done with "gcc", the GNU
-   project's excellent C compiler.  Releases can be compiled with gcc
-   or most other compilers (such as the unbundled Solaris C compiler).
+Development of Freeciv21 is primarily done with "gcc", the GNU project's excellent C compiler, or on Windows MS Visual C.
```

Comment:
MSVC isn't working yet, also see comment above.

Reply:
Fixed in third commit